mubench.reviewsite/init_db.sql

```diff
@@ -83,6 +83,23 @@ CREATE TABLE IF NOT EXISTS `types` (
   UNIQUE KEY `name` (`name`)
 ) ENGINE=MyISAM  DEFAULT CHARSET=latin1;
 
+CREATE TABLE IF NOT EXISTS `misuse_tags` (
+  `exp` varchar(10) NOT NULL,
+  `detector` varchar(100) NOT NULL,
+  `project` varchar(100) NOT NULL,
+  `version` varchar(100) NOT NULL,
+  `misuse` varchar(100) NOT NULL,
```

Comment:
Note that I reduced the max length of detector to 10 (it should be an int anyways...) and of project, version, and misuse to 30, because the database on my production server complained about "key too large" otherwise. I thing that these sizes are sufficient, so please use them here, too.